calcom/cal.com
# feat: add optimistic update for away status toggling

## What does this PR do?

Adds optimistic update for away status toggling

Fixes #7749 





## Type of change




## How should this be tested?



Just open dropdown user menu and toggle your away status. It should be toggled instantly/optimistically. If there is an error then the toast will be shown.

## Mandatory Tasks


/claim #7749
